### DISCUSSION TITLE
How to register a user using the admin API

### DISCUSSION THREAD

**Discussion by droslean** [keycloak/keycloak#23782] [Q&A] [answered]
https://www.keycloak.org/docs-api/22.0.1/rest-api/#_users is unclear. Searching the web, I found many articles that explain how to send a POST to the ` /admin/realms/{realm}/users` endpoint but this endpoint is completely missing, for POST, in the docs.

Does any documentation explain how users are being added using the API? How the user actions are being managed, etc.

Thanks in advanced.

**Comment by kwame-mintah** [comment, marked as answer]
@droslean, the POST documentation is present. It's just further down the page, try searching for this text on the page:

> POST /admin/realms/{realm}/users
> Create a new user Username must be unique. 

<img width="1011" alt="Screenshot 2023-10-09 at 10 14 39" src="https://github.com/keycloak/keycloak/assets/37197235/e16f0e55-3ad0-4d8b-834f-dd7eebeb8316">
